website/content/en/docs/setup/installation/platforms/kubernetes.md

```diff
@@ -20,14 +20,10 @@ You can install Vector on Kubernetes using either [Helm](#helm) or [kubectl](#ku
 
 ### kubectl
 
-[kubectl] is the Kubernetes command-line tool. You can use it as an alternative to [Helm](#helm) to install Vector on Kubernetes The instructions below are for installing Vector in the [agent] role.
-
-{{< warning title="Aggregator role in public beta" >}}
-Support for the [aggregator] role is currently in public beta. We're seeking beta testers! If deploying the aggregator, please [join our chat][chat] and let us know how it went.
+[kubectl] is the Kubernetes command-line tool. You can use it as an alternative to [Helm](#helm) to install Vector on Kubernetes The instructions below are for installing Vector in the [agent] and [aggregator] roles.
 
+[agent]: /docs/setup/deployment/roles/#agent
 [aggregator]: /docs/setup/deployment/roles/#aggregator
```

Comment:
Since we seem to make a point of capitalizing `Agent` and `Aggregator` elsewhere, do we want to do so here?

Reply:
Those specifically are just link specifications (I think). I'll double check the the rest of the file for missing capitalizations in the actual content 👍